js/viewer/annotation/annotation-svg.js

```diff
@@ -165,17 +165,26 @@ function AnnotationSVG(parent, id, imgWidth, imgHeight, scale, ignoreReferencePo
             .remove();
     }
 
+    // get viewbox of the svg 
+    this.getViewBox = function () {
+        return (this.viewBox) ? this.viewBox : [0, 0, this.imgWidth, this.imgHeight];
+    }
+
     this.parseSVGFile = function(svgFile){
 
         if(!svgFile){ return; }
 
-        this.viewBox = svgFile.getAttribute("viewBox").split(" ").map(
-            function(num){ 
-                return +num;
-            }
-        );
+        if (svgFile.getAttribute("viewBox")) {
```

Comment:
You should explain the viewbox logic somewhere in this code. Either in the jsdoc description of this function or as part of an inline comment here.